### PR TITLE
s-t integrations: apply sentence-style caps to categories

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

s-t integrations: 
- apply sentence-style caps to categories

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/sisyphus.markdown [hunk 1]
```diff
@@ -21,7 +21,7 @@ The [Sisyphus](https://sisyphus-industries.com/) integration for Home Assistant
 There is currently support for the following device types within Home Assistant:
 
 - **Light** - Can be used to sleep/wake the table or adjust the brightness of the table light.
-- **Media Player** - Can be used to sleep/wake the table, play/pause, move between tracks, or toggle shuffle on and off. The "volume" control adjusts the table's speed.
+- **Media player** - Can be used to sleep/wake the table, play/pause, move between tracks, or toggle shuffle on and off. The "volume" control adjusts the table's speed.
 
 The Light and Media players will be automatically added for each of your Sisyphus tables, if the Sisyphus integration is configured.
```
Comment (ludeeus) [inline source/_integrations/sisyphus.markdown]:
```suggestion
The light and media players will be automatically added for each of your Sisyphus tables, if the Sisyphus integration is configured.
```

File: source/_integrations/smartthings.markdown [hunk 2]
```diff
@@ -136,9 +136,9 @@ SmartThings represents devices as a set of [capabilities](https://developer-prev
 
 Support for additional platforms will be added in the future.
 
-### Binary Sensor
+### Binary sensor
 
-The SmartThings Binary Sensor platform lets you view devices that have binary sensor-related capabilities. A Binary Sensor entity will be created for each attribute (below) supported by the device.
+The SmartThings Binary sensor platform lets you view devices that have binary sensor-related capabilities. A Binary sensor entity will be created for each attribute (below) supported by the device.
```
Comment (ludeeus) [inline source/_integrations/smartthings.markdown]:
```suggestion
The SmartThings binary sensor platform lets you view devices that have binary sensor-related capabilities. A binary sensor entity will be created for each attribute (below) supported by the device.
```

File: source/_integrations/template.markdown [hunk 3]
```diff
@@ -648,7 +648,7 @@ template:
 
 {% endraw %}
 
-### State based binary sensor - Device Tracker sensor with Latitude and Longitude Attributes
+### State based binary sensor - Device tracker sensor with Latitude and Longitude Attributes
```
Comment (ludeeus) [inline source/_integrations/template.markdown]:
```suggestion
### State based binary sensor - device tracker sensor with latitude and longitude attributes
```

File: source/_integrations/totalconnect.markdown [hunk 4]
```diff
@@ -76,9 +76,9 @@ sudo apt install libxml2-dev libxmlsec1-dev
 
 {% enddetails %}
 
-## Alarm Control Panel
+## Alarm control panel
 
-The integration provides an Alarm Control Panel for each Total Connect location. It uses the name of your location from Total Connect.  For example, if your location name in Total Connect is "Home", Home Assistant will use `alarm_control_panel.home`.
+The integration provides an Alarm control panel for each Total Connect location. It uses the name of your location from Total Connect.  For example, if your location name in Total Connect is "Home", Home Assistant will use `alarm_control_panel.home`.
```
Comment (ludeeus) [inline source/_integrations/totalconnect.markdown]:
```suggestion
The integration provides an alarm control panel for each Total Connect location. It uses the name of your location from Total Connect.  For example, if your location name in Total Connect is "Home", Home Assistant will use `alarm_control_panel.home`.
```

File: source/_integrations/sisyphus.markdown (hunk 1 above)
Comment (klaasnicolaas) [reply source/_integrations/sisyphus.markdown]: Out of scope for this PR 🤔

File: source/_integrations/shopping_list.markdown [hunk 5]
```diff
@@ -16,7 +16,7 @@ ha_platforms:
 
 The `shopping_list` integration allows you to keep track of shopping list items. 
 
-Your shopping list will be accessible from the sidebar, and you can optionally add the [Shopping List card](/dashboards/shopping-list/) to your dashboard. With the [Conversation integration](/integrations/conversation/) you can add items to your shopping list using voice commands like "Add eggs to my shopping list." 
+Your shopping list will be accessible from the sidebar, and you can optionally add the [shopping list card](/dashboards/shopping-list/) to your dashboard. With the [Conversation integration](/integrations/conversation/) you can add items to your shopping list using voice commands like "Add eggs to my shopping list."
```
Comment (joostlek) [inline source/_integrations/shopping_list.markdown]:
```suggestion
Your shopping list will be accessible from the sidebar, and you can optionally add the [shopping list card](/dashboards/shopping-list/) to your dashboard. With the [conversation integration](/integrations/conversation/) you can add items to your shopping list using voice commands like "Add eggs to my shopping list." 
```
👀

File: source/_integrations/simplisafe.markdown [hunk 6]
```diff
@@ -26,7 +26,7 @@ The `simplisafe` integration integrates [SimpliSafe home security](https://simpl
 
 There is currently support for the following device types within Home Assistant:
 
-- **Alarm Control Panel**: reports on the current alarm status and can be used to arm and disarm the system.
+- **Alarm control panel**: reports on the current alarm status and can be used to arm and disarm the system.
 - **CO Detector**: reports on the carbon monoxide sensor status*.
```
Comment (joostlek) [inline source/_integrations/simplisafe.markdown]:
```suggestion
- **CO detector**: reports on the carbon monoxide sensor status*.
```

File: source/_integrations/simplisafe.markdown [hunk 7]
```diff
@@ -26,7 +26,7 @@ The `simplisafe` integration integrates [SimpliSafe home security](https://simpl
...
 - **Entry Sensor**: reports on the current entry sensor status*.
 - **Freeze Sensor**: reports on the freeze sensor temperature*.
```
Comment (joostlek) [inline source/_integrations/simplisafe.markdown]:
```suggestion
- **Entry sensor**: reports on the current entry sensor status*.
- **Freeze sensor**: reports on the freeze sensor temperature*.
```

File: source/_integrations/starline.markdown [hunk 8]
```diff
@@ -27,7 +27,7 @@ The `starline` integration lets you retrieve data of your [StarLine](https://www
 
 This integration provides the following platforms:
 
-- Binary Sensors: Hand brake, hood, trunk, alarm status and doors lock state.
+- Binary sensors: Hand brake, hood, trunk, alarm status and doors lock state.
 - Device tracker: The location of your car.
 - Lock: Control the lock of your car.
 - Sensors: Battery level, SIM card balance, GSM signal level, Fuel Volume, Mileage, OBD Errors, interior temperature and engine temperature.
```
Comment (joostlek) [inline source/_integrations/starline.markdown]:
```suggestion
- Sensors: Battery level, SIM card balance, GSM signal level, fuel volume, mileage, OBD errors, interior temperature and engine temperature.
```

**Review by joostlek — DISMISSED** [review]

**Review by joostlek — APPROVED** [review]

**Pull request merged**